wts/api.py

```diff
@@ -113,21 +121,20 @@ def _log_and_jsonify_exception(e):
 app.register_error_handler(APIError, _log_and_jsonify_exception)
 
 
-@app.before_first_request
```

Comment:
So we were running into these errors with the unit tests:
```
======================================================================================= short test summary info ========================================================================================
FAILED tests/app_test.py::test_connected_endpoint - ValueError: The name 'oauth2' is already registered for this blueprint. Use 'name=' to provide a unique name.
FAILED tests/app_test.py::test_token_endpoint_with_default_idp - ValueError: The name 'oauth2' is already registered for this blueprint. Use 'name=' to provide a unique name.
FAILED tests/app_test.py::test_token_endpoint_with_idp_a - ValueError: The name 'oauth2' is already registered for this blueprint. Use 'name=' to provide a unique name.
FAILED tests/app_test.py::test_token_endpoint_without_specifying_idp - ValueError: The name 'oauth2' is already registered for this blueprint. Use 'name=' to provide a unique name.
FAILED tests/app_test.py::test_token_endpoint_with_bogus_idp - ValueError: The name 'oauth2' is already registered for this blueprint. Use 'name=' to provide a unique name.
FAILED tests/app_test.py::test_token_endpoint_without_auth_header - ValueError: The name 'oauth2' is already registered for this blueprint. Use 'name=' to provide a unique name.
FAILED tests/app_test.py::test_authorize_endpoint - ValueError: The name 'oauth2' is already registered for this blueprint. Use 'name=' to provide a unique name.
FAILED tests/app_test.py::test_authorization_url_endpoint - ValueError: The name 'oauth2' is already registered for this blueprint. Use 'name=' to provide a unique name.
FAILED tests/app_test.py::test_external_oidc_endpoint - ValueError: The name 'oauth2' is already registered for this blueprint. Use 'name=' to provide a unique name.
FAILED tests/system_test.py::test_status_endpoint - ValueError: The name 'oauth2' is already registered for this blueprint. Use 'name=' to provide a unique name.
===================================================================================== 10 failed, 2 passed in 3.77s =====================================================================================
```
And it turned out out that the unit tests attempted to register the app blueprints twice:
- by the app before the first request because of the above decorator
- [in conftest.py](https://github.com/uc-cdis/workspace-token-service/blob/3d9afdc226ba09014274c38cb46fda5fec864316/tests/conftest.py#L40)

Not exactly clear why, but we think the upgrade to Python 3.9 surfaced this (e.g. these errors aren't occurring in master which is 3.6 and also didn't occur last year when using 3.7 for this branch).

https://github.com/uc-cdis/workspace-token-service/pull/42